scrubber/scrubUtils.ts

```diff
@@ -0,0 +1,163 @@
+import fs from "fs";
+import path from "path";
+import { ScrubberAction, TagNameToAction } from "./scrubberTypes";
+
+const TAG_START_CHAR = "{";
+const TAG_END_CHAR = "}";
+
+const FILE_TYPE_COMMENT: { [key: string]: string } = {
+  js: "//",
```

Comment:
Should we account for jsx and tsx as well?

Reply:
Good catch -- going to add this in another PR